src/game_system.cpp

```diff
@@ -187,6 +187,17 @@ void Game_System::SePlay(RPG::Sound const& se, bool stop_sounds) {
 	request->Start();
 }
 
+void Game_System::SePlay(const RPG::Animation &animation) {
+	std::string path;
+	std::vector<RPG::AnimationTiming>::const_iterator it = animation.timings.begin();
+	for (; it != animation.timings.end(); ++it) {
+		if (!isStopFilename(it->se.name, FileFinder::FindSound, path) && !path.empty()) {
```

Comment:
this can be replaced with a for(const auto& anim)-loop

The path.empty check is not necessary, isStopFilename already ensures non-empty paths when returning true